src/templates/product.html

```diff
@@ -66,5 +66,52 @@ <h1 class="page-header-title">{{ title | safe }}</h1>
 
     </main>
 
+    <script>
+        (function() {
+            if (!window.Promise) {
+                var script = document.createElement('script');
+                script.src = 'https://rawgit.com/voorhoede/voorhoede-ocelot-formatter/fonts/src/assets/scripts/promise.js';
+                document.head.appendChild(script);
+            }
+        }());
+    </script>
+    <script>
+        (function(window) {
+            var html = document.documentElement;
+            var FONTS_LOADED_CLASS = 'fonts-loaded';
+
+            if (sessionStorage.fontsLoaded) {
```

Comment:
move the `fontsLoaded` check to the `<head>` to prevent the FOUT on repeat views.